src/udev/udev-builtin-net_id.c

```diff
@@ -775,6 +775,14 @@ static int builtin_net_id(struct udev_device *dev, int argc, char *argv[], bool
         if (!streq(s, p))
                 return 0;
 
+        /* Skip Libertas devices with an OLPC mesh companion */
+        s = udev_device_get_sysattr_value(dev, "lbs_mesh");
+        if (s && streq(s, "0x1"))
+                return 0;
+        s = udev_device_get_sysattr_value(dev, "anycast_mask");
+        if (s)
+                return 0;
```

Comment:
hmm, I think it would make sense to maybe suffix one of the two names with something (maybe `m` for mesh), and stick to the naming.

I mean, whether the naming is "silly" as you say or not is not really the point here. if people think so they should turn off the naming altogether, but we shouldn't do that for them implicitly. It's up to downstreams/users to pick the naming scheme, in code we should try to support something roughly reasonable if we can, and afaics we can...

Also, I'd prefer if the handling of this bit could be split out into a function of its own, carrying a comment what this is about.